examples/api-routes-apollo-server-and-client/apollo/type-defs.js

```diff
@@ -0,0 +1,7 @@
+import gql from 'graphql-tag'
+
+export default gql`
```

Comment:
Same as with the resolvers, remove this file and move the schema to `init-apollo.js`, then you can remove the `apollo` folder, that will reduce the complexity of the example